setup.py

```diff
@@ -24,8 +24,7 @@
 ]
 
 extras_require = {
-    "format": ["rfc3987", "strict-rfc3339", "webcolors"],
-    ":python_version=='2.7'": ["functools32"],
+    "format": ["rfc3987", "strict-rfc3339", "webcolors"]
```

Comment:
I was wondering if we can move everything out of `extra_require` but I could not find in any documentation the proper syntax for this `format` key.

Any hint?

Reply:
Don’t think so. The `format` key is using `extras_require` as it is originally designed to do, i.e. support the `package[extra]` installation syntax. There are no alternatives to this feature. `extras_require` itself is not obsolete, only using it for environment markers is.